src/component.js

```diff
@@ -47,7 +47,10 @@ function createSubmitHandler (hostedFieldsInstance, orderIdFunction) : Function
           hosted_payment_session_cre_ts_epoch: Date.now().toString()
         });
 
-        return { orderId };
+        return {
+          liabilityShifted: payload.success,
```

Comment:
Is `liabilityShifted` what y'all want this to say here?

Reply:
In the case of 3ds being skipped (because the card is not eligible), what should we pass here?

Should we only pass a `liabilityShifted` param if it goes through the 3ds flow?